src/pages/index.jsx

```diff
@@ -187,7 +244,7 @@ export default function Home({ episodes }) {
           </div>
```

Comment:
For the `EpsiodeSearchBar` rename to prop `searchClicked` to `showSearchBar` or something and the value of it should be `query.length > 2 || searchClicked`